CHANGELOG_DESKTOP.md

```diff
@@ -1,5 +1,50 @@
 # Changelog
 
+## [1.7.92](https://github.com/brave/brave-browser/releases/tag/v1.7.92)
+
+ - Added Monthly Statements for Brave Rewards. ([#930](https://github.com/brave/brave-browser/issues/930))
```

Comment:
change to lowercase `monthly statements`

Reply:
updated